tests/endtoend/test_file_name_functions.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+from unittest.mock import patch
+
+import requests
+
+from azure_functions_worker.constants import SCRIPT_FILE_NAME
+from tests.utils import testutils
+
+REQUEST_TIMEOUT_SEC = 5
+
+
+class TestHttpFunctionsFileName(testutils.WebHostTestCase):
```

Comment:
Are these tests copied over from anywhere else, or created only for this test?

Reply:
Since it's a new app setting, I created a separate test file for it (similarly to how the other app settings were tested like PTPTC). The tests are based off of test_http_functions.py, since the new function app I added has 2 functions with http triggers